vertx-web/src/main/java/io/vertx/ext/web/impl/Utils.java

```diff
@@ -40,93 +42,254 @@
   private static final Pattern SEMICOLON_SPLITTER = Pattern.compile(" *; *");
   private static final Pattern EQUAL_SPLITTER = Pattern.compile(" *= *");
 
-  public static String normalisePath(String path) {
```

Comment:
That is in the impl package so it should not matter, it should not break anything for normal usages, however for paths like: `/foo..bar` it will break because the old non RFC compliant algorithm would return `/foobar` which is wrong it should be `/foo..bar` (This was the case that was in the bug report).